.github/scripts/setup_heroku_instance.sh

```diff
@@ -41,7 +41,7 @@ resultOfServerPush=$?
 resultOfSeedDataSetup=0
 if [ $serverAppAlreadyExists = false ]; then
   echo "Setting up initial seed data"
-  (cd $serverAppDirectory && heroku run rails db:schema:load db:seed)
+  (cd $serverAppDirectory && heroku run rails db:structure:load:with_data db:seed)
```

Comment:
Why did the command change? Is it a path to setting up server side heroku instance?

Reply:
Yes, this sets up the server side database schema with seed data. This was something that was changed on the server side setup recently, so we needed to update our local setup script to make things work again.